### ISSUE TITLE
Intense amounts of damage randomly applied to entities.

### ISSUE THREAD

**Issue by Alexxithymla** [studio-minus/ppg-bugs-and-requests#2396]
**Describe the bug**
Randomly when I'm moving around one of my people they take a lot of damage randomly, all their body gets like crushed and stuff.

**Clearly describe how to reproduce the bug**
I'm not too sure, just randomly happens when I move around my people or when they touch each other or another object.

[exported game debug info.md](https://github.com/studio-minus/ppg-bugs-and-requests/files/10000717/exported.game.debug.info.md)

**Additional notes**

**Comment by royalealbert27** [comment]
epic unity physics

**Comment by mestiez** [comment]
yeah i tried to compensate for this by taking all the impulse values provided by the physics engine, removing outliers, and using the average. it helps but it doesnt completely solve the issue
